Open-vm-tools fails to install due to missing gdk-pixbuf-xlib-2.0

**Describe the bug**
Open-vm-tools fails to install:

```
checking for gdk-pixbuf-xlib-2.0 >= 2.21.0 (via pkg-config)... no
configure: error: gdk-pixbuf-xlib-2.0 not found. Please configure without X11 (using --without-x) or install the gdk-pixbuf-xlib-2.0 devel package.
builder for '/nix/store/aryy4mq9nr08dlvmhb2prm6zpyk2dmzr-open-vm-tools-11.0.5.drv' failed with exit code 1
```

**To Reproduce**
```
virtualisation.vmware.guest.enable = true;
```
in /etc/nixos/configuration.nix

**Notify maintainers**
joamaki 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.44, NixOS, 20.09pre228622.029a5de0839 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(alex): `"home-manager"`
 - channels(root): `"nixos-20.09pre229574.0a146054bdf, home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: open-vm-tools
# a list of nixos modules affected by the problem
module: virtualisation.vmware.guest
```

--- a/pkgs/applications/virtualization/open-vm-tools/default.nix
+++ b/pkgs/applications/virtualization/open-vm-tools/default.nix
@@ -1,7 +1,7 @@
 { stdenv, lib, fetchFromGitHub, makeWrapper, autoreconfHook,
   fuse, libmspack, openssl, pam, xercesc, icu, libdnet, procps,
   libX11, libXext, libXinerama, libXi, libXrender, libXrandr, libXtst,
-  pkgconfig, glib, gtk3, gtkmm3, iproute, dbus, systemd, which,
+  pkgconfig, glib, gdk-pixbuf-xlib, gtk3, gtkmm3, iproute, dbus, systemd, which,
   withX ? true }:
 
 stdenv.mkDerivation rec {
@@ -21,9 +21,13 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook makeWrapper pkgconfig ];
   buildInputs = [ fuse glib icu libdnet libmspack openssl pam procps xercesc ]
-      ++ lib.optionals withX [ gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
+      ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
-  patches = [ ./recognize_nixos.patch ];
+  patches = [ 
+    ./recognize_nixos.patch 
+    ./find_gdk_pixbuf_xlib.patch #See https://github.com/vmware/open-vm-tools/pull/438
+  ];
+
   postPatch = ''
      # Build bugfix for 10.1.0, stolen from Arch PKGBUILD
      mkdir -p common-agent/etc/config

--- a/pkgs/applications/virtualization/open-vm-tools/find_gdk_pixbuf_xlib.patch
+++ b/pkgs/applications/virtualization/open-vm-tools/find_gdk_pixbuf_xlib.patch
@@ -1,0 +1,12 @@
+diff --git a/lib/appUtil/Makefile.am b/lib/appUtil/Makefile.am
+index a0d8e391..899cd4e9 100644
+--- a/lib/appUtil/Makefile.am
++++ b/lib/appUtil/Makefile.am
+@@ -21,4 +21,6 @@ libAppUtil_la_SOURCES =
+ libAppUtil_la_SOURCES += appUtil.c
+ libAppUtil_la_SOURCES += appUtilX11.c
+ 
+-AM_CFLAGS = @GTK_CPPFLAGS@
++AM_CFLAGS =
++AM_CFLAGS += @GTK_CPPFLAGS@
++AM_CFLAGS += @GDK_PIXBUF_XLIB2_CPPFLAGS@

